lib/datumUtils.js

```diff
@@ -3,21 +3,21 @@ var PJD_3PARAM = 1;
 var PJD_7PARAM = 2;
 var HALF_PI = Math.PI/2;
 
-exports.compareDatums = function(source, dest) {
+export function compareDatums(source, dest) {
   if (source.datum_type !== dest.datum_type) {
     return false; // false, datums are not equal
-  } else if (source.a !== dest.a || Math.abs(this.es - dest.es) > 0.000000000050) {
-    // the tolerence for es is to ensure that GRS80 and WGS84
+  } else if (source.a !== dest.a || Math.abs(source.es - dest.es) > 0.000000000050) {
+    // the tolerance for es is to ensure that GRS80 and WGS84
     // are considered identical
     return false;
   } else if (source.datum_type === PJD_3PARAM) {
-    return (this.datum_params[0] === dest.datum_params[0] && source.datum_params[1] === dest.datum_params[1] && source.datum_params[2] === dest.datum_params[2]);
+    return (source.datum_params[0] === dest.datum_params[0] && source.datum_params[1] === dest.datum_params[1] && source.datum_params[2] === dest.datum_params[2]);
```

Comment:
that is a good call and I'm actually going to get that in a separate patch to get it in without waiting for this one